automations/profilefollower.py

```diff
@@ -96,8 +95,8 @@ def execute(self):
             self.position_error_i += pos_error
 
             # generate the linear output to the chassis (m/s)
-            linear_output = (self.kP * pos_error + self.kV * linear_seg[1]
-                + self.kA * linear_seg[2] + self.kI*self.position_error_i
+            linear_output = (self.kP*pos_error + self.kV*linear_seg[1]
+                + self.kA*linear_seg[2] + self.kI*self.position_error_i
```

Comment:
Could be useful to others. If we remove the kA term, we should remove all the code related to it as well — this would mean not returning acceleration from the profile generators.